src/vehicle_cmd.cpp

```diff
@@ -312,16 +312,16 @@ static CommandCost RefitVehicle(Vehicle *v, bool only_this, uint8 num_vehicles,
 		v = v->First();
 	}
 
-	static SmallVector<RefitResult, 16> refit_result;
-	refit_result.Clear();
+	static std::vector<RefitResult> refit_result;
```

Comment:
i wonder what the purpose of this vector being static is, if it's just cleared on the next line anyway...

Reply:
True but probably beyond the scope of this PR?

Reply:
Done anyway